app/controllers/flash_teams_controller.rb

```diff
@@ -25,12 +25,12 @@ def new
     @task_status = Hash.new
```

Comment:
Seems like most of the changes in this file are whitespace. Could you take it out of the PR?